syntax_and_semantics/class_methods.md

```diff
@@ -0,0 +1,33 @@
+# Class methods
+
+Class methods are methods associated to a class or module instead of a specific instance.
+
+```crystal
+module CaesarCipher
+  def self.encrypt(string : String)
+    string.chars.map{ |char| ((char.upcase.ord - 52) % 26 + 65).chr }.join
+  end
+end
+
+CaesarCipher.encrypt("HELLO") # => "URYYB"
+```
+
+Class methods are defined by prefixing the method name with the type name and a period.
+
+```
```

Comment:
Any reason not to use syntax highlighting on this code block? It looks strange, especially given the block above does highlight.

Reply:
No reason, just didn't make it into my fingers 😄